### PR TITLE
Dev 58 - improved layout of the UpdateUserProfile page

### PR DESCRIPTION
**Pull request by KovDimaY**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure all the tests are still passing.
* [x] Make sure the test coverage is still 100%.
* [x] Make sure your source branch is correct.
* [x] Make sure your destination branch is **devel**.
* [x] Make sure the name of the PR has a correct DEV-# identification prefix.
* [x] Make sure you reviewed all your changes and they look correct.
* [x] Make sure all the commits have a correct format. 
* [x] Make sure you refer related issues for you PR. 
* [x] Make sure you add **KovDimaY** as a reviewer and assignee of your PR.

Take a look at the Issue #105

### CODE REVIEW

File: client/src/containers/EditUserProfile/index.js [hunk 1]
```diff
@@ -121,6 +123,17 @@ class EditUserProfile extends PureComponent {
...
   }
 
+  handleCountryChange = (countryObject) => {
+    const newFormdata = {
+      ...this.state.formData,
+      nationality: countryObject.countryName,
+    };
+
+    this.setState({
+      formData: newFormdata,
```
Comment (KovDimaY) [inline client/src/containers/EditUserProfile/index.js]: Makes no sense to have it written using 3 lines

File: client/src/containers/EditUserProfile/styles.css [hunk 2]
```diff
@@ -1,15 +1,64 @@
...
+    color: black;
+    border-radius: 0;
+}
+.edit-user-profile-container .info .country-selector-container .c-dropdown-menu {
+    font-size: 16px;
+    font-weight: normal;
+    color: black;
+}
+
+
```
Comment (KovDimaY) [inline client/src/containers/EditUserProfile/styles.css]: Remove extra spaces left from development

File: client/src/containers/EditUserProfile/styles.css [hunk 3]
```diff
@@ -18,21 +59,12 @@
...
-    display: block;
-    color: orange;
-}
 .edit-user-profile-container .danger-zone {
     color: orangered;
     text-align: center;
     font-size: 24px;
     font-weight: bold;
-    margin-bottom: 0;
+    margin-bottom: 4;
```
Comment (NikolayYakovenko) [inline client/src/containers/EditUserProfile/styles.css]: You missed **px** unit.

File: client/src/containers/EditUserProfile/styles.css [hunk 4]
```diff
@@ -109,4 +167,18 @@
...
+    .edit-user-profile-container .form-content-wrapper {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+    .edit-user-profile-container .left-column,
+    .edit-user-profile-container .right-column {
+        width: 75%;
+        margin: auto;
+        max-width: 400px;
```
Comment (NikolayYakovenko) [inline client/src/containers/EditUserProfile/styles.css]: I am not sure, but `width: 75%;` is not necessary here. You can just leave `max-width: 400px;`

File: client/src/components/User/styles.css [hunk 5]
```diff
@@ -1,39 +1,80 @@
-
-.user_container {
-    margin-bottom: 75px;
+.user-container {
```
Comment (NikolayYakovenko) [inline client/src/components/User/styles.css]: There is a "problem" in CSS with parent of floated blocks.
If you use **float** for some element in your layout, you should use **clear** to cancel floating for the next element.
http://htmlbook.ru/css/clear
You can check this problem in DevTools. Hover on .user-container. Its height would be less the then it looks on the page.
To fix this you can use one of possible solutions:

- The simple and faster: add `overflow: hidden;` to **.user-container**
- Add this styles (it is often called like clearfix :smile:):
```
.user-container:after {
    content: '';
    clear: both;
    display: table;
}
```

File: client/src/containers/EditUserProfile/styles.css (hunk 3 above)
Comment (KovDimaY) [reply client/src/containers/EditUserProfile/styles.css]: Thank you for the attention! :)

File: client/src/components/User/styles.css (hunk 5 above)
Comment (KovDimaY) [reply client/src/components/User/styles.css]: Thank you! Finally I decided to do not use float, but it is good to know for the future! ❤️

**Pull request merged**